apis/actions.github.com/v1alpha1/autoscalinglistener_types.go

```diff
@@ -57,6 +57,9 @@ type AutoscalingListenerSpec struct {
 
 	// +optional
 	Proxy *ProxyConfig `json:"proxy,omitempty"`
+
+	// +optional
+	GitHubServerTLS *GitHubServerTLSConfig `json:"githubServerTLS,omitempty"`
```

Comment:
What's the rational for calling it `GitHubServerTLS` as opposed to something more generic?

Reply:
This was already part of the spec in other types, it's being discussed [here](https://github.com/actions/actions-runner-controller/pull/2268#discussion_r1123110932) if you don't mind moving the conversation there 🙇